hedera-node/src/main/java/com/hedera/services/ledger/CommitInterceptor.java

```diff
@@ -36,7 +36,8 @@
 @FunctionalInterface
 public interface CommitInterceptor<K, A, P extends Enum<P> & BeanProperty<A>> {
 	/**
-	 * Accepts a pending change set to preview before it is committed to the ledger.
+	 * Accepts a pending change set to preview before it is committed to the ledger. Builds list of first order
+	 * reward situations if staking is enabled.
```

Comment:
This comment would be better suited to `StakeAwareAccountsCommitsInterceptor.preview()` alone, since there are several `CommitInterceptor` implementations that have nothing to do with staking.

Reply:
Removed staking information from the comment